flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/catalog/CatalogTestBase.java

```diff
@@ -168,7 +168,7 @@ protected Map<String, String> getBatchTableProperties() {
         return new HashMap<String, String>() {
             {
                 put(IS_STREAMING, "false");
-                putAll(getGenericFlag(isGeneric()));
+                put(FactoryUtil.CONNECTOR.key(), "hive");
```

Comment:
Can we move the hive specific reference into a hive module as part of this change? I assume setting this as hive is misleading and not useful for a non-hive use case.